src/nvim/CMakeLists.txt

```diff
@@ -65,6 +65,10 @@ foreach(sfile ${NEOVIM_SOURCES})
   if(${f} MATCHES "^(regexp_nfa.c)$")
```

Comment:
@tarruda What is the reason for the special handling of `regexp_nfa.c` in this cmake file?